src/main/kotlin/org/pkl/intellij/util/Util.kt

```diff
@@ -196,8 +196,8 @@ val AnnotationHolder.currentFile: PsiFile
 val AnnotationHolder.currentProject: Project
   get() = currentFile.project
 
-val AnnotationHolder.currentModule: PklModule
-  get() = currentFile as PklModule
+val AnnotationHolder.currentModule: PklModule?
+  get() = currentFile as? PklModule
```

Comment:
Unrelated: this was causing a null pointer exception because `getCurrentFile()` can possibly return something that is not a `PklModule`.